processing/src/main/java/org/apache/druid/java/util/metrics/OshiSysMonitor.java

```diff
@@ -313,8 +313,9 @@ public void emit(ServiceEmitter emitter)
         for (String addr : net.getIPv4addr()) {
           if (!NET_ADDRESS_BLACKLIST.contains(addr)) {
             // Only emit metrics for non black-listed ip addresses
+            String mapKey = name + "_" + addr;    // Network_Name_IPV4 address as key, ex: wifi_192.1.0.1 to uniquely identify the dimension
```

Comment:
looks like the tests were indeed helpful :)

Reply:
Yes they were 😅